### ISSUE TITLE
Screensaver not receiving input after suspending

### ISSUE THREAD

**Issue by aerkiaga** [linuxmint/cinnamon-screensaver#285]
```
 * cinnamon-screensaver 3.8.2
 * Linux Mint 19
 * GeForce GT 730, nvidia-driver-390 390.48-0ubuntu3
 * 64 bit
 ```

**Issue**
After suspending the computer, I can wake it up by pressing a key on my keyboard (USB keyboard), and the screensaver appears. I am prompted to type my password, the mouse cursor (USB mouse) and the keyboard layout icon appear, but I can't type anything or move the cursor. The screensaver ends up hiding the text box out of inactivity, and it doesn't respond to any input, even after reconnecting both peripherals.

**Steps to reproduce**
Happens every time I suspend my computer.

**Expected behaviour**
It should receive input and react accordingly. Typing on the keyboard should insert text in the password box, and moving the mouse should produce the corresponding motion on the pointer.

**Other information**
Output from lsusb:
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 006 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 002 Device 002: ID 148f:5372 Ralink Technology, Corp. RT5372 Wireless Adapter
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 005 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 003: ID 0e6a:02c1 Megawin Technology Co., Ltd 
Bus 004 Device 002: ID 275d:0ba6  
Bus 004 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 008 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 007 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

**Comment by icarter09** [comment]
@aerkiaga are you still experiencing this problem?  

@CottonEaster are you still experiencing this problem as well?

**Comment by CottonEaster** [comment]
No, I have not seen this for a while, I am happy to say.

**Comment by icarter09** [comment]
@aerkiaga if you are no longer experiencing this problem, can you close this ticket out? Thanks.

**Comment by aerkiaga** [comment]
Sorry to say... But it still happens consistently every time I suspend my computer.

**Comment by icarter09** [comment]
@aerkiaga can you provided the steps for how you are putting your computer in suspend mode? Thanks.

**Comment by aerkiaga** [comment]
Of course. I open the Start Menu, click on 'Quit', then select 'Suspend' in the dialog box. The computer becomes idle just as expected, and the screen lights up again when keyboard input is received, but no further input is processed, until the screensaver starts with no way to stop it. I will try other keyboards/mice just to see if the problem is specific to some particular model.

**Comment by icarter09** [comment]
@aerkiaga thanks!

Does running `systemctl suspend` from the terminal give you the same problem?

**Comment by icarter09** [comment]
Thanks for that update. Sounds like we are getting closer to finding the root of the problem.

**Comment by aerkiaga** [comment]
Some other info that could be useful?

**Comment by aerkiaga** [comment]
Ok, this keeps happening even after upgrading to Linux Mint 19.1. Could it be a driver issue?
